packages/rich-text/src/plugins/Normalizer/baseRules.ts

```diff
@@ -34,4 +35,14 @@ export const baseRules: Required<NormalizerRule>[] = [
     },
     transform: transformWrapIn(BLOCKS.PARAGRAPH),
   },
+  {
+    // Replaces `\r` as it causes issues with scroll position and focus
+    match: isText,
+    validNode: (_editor, [node]) => typeof node.text === 'string' && !node.text.includes('\r'),
+    transform: (editor, [node, path]) => {
+      if (typeof node.text === 'string') {
```

Comment:
Should we remove this if since the check is handled in the predicate? otherwise the callback here theoretically doesn't always return a value.

Reply:
in theory yes, we could then to do a type cast to satisfy typescript